osquery/requests/serializers/json_tests.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright (c) 2014, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <gtest/gtest.h>
+
+#include "osquery/requests/serializers/json.h"
+
+namespace osquery {
+
+class JSONTests : public testing::Test {};
+
+TEST_F(JSONTests, testSerialize) {
```

Comment:
Yeah, but I still feel like it should be there, what if we want to use a different implementation one day? This is a bit of a contrived serializer here, mostly because JSON is so well supported. Having a standard of how serializers are tested, that they're tested, is good, I think.